generators/typescript-v2/ast/src/ast/Type.ts

```diff
@@ -18,8 +105,58 @@ export class Type extends AstNode {
             case "string":
                 writer.write("string");
                 break;
+            case "number":
+                writer.write("number");
+                break;
+            case "bigint":
+                writer.write("bigint");
```

Comment:
Let's add a TODO here since this will actually require using some configuration. We only support `bigInt` when we've actually configured the generator to do so ([link](https://github.com/fern-api/fern/blob/0c3d46efacba93049a6485cce44394955c0eac5d/generators/typescript/sdk/cli/src/SdkGeneratorCli.ts#L64)).

Reply:
can do